packages-backend/express/README.md

```diff
@@ -26,20 +26,58 @@ const {
   CLOUD_IDENTIFIERS,
 } = require('@commercetools-backend/express');
 
-app.use(createSessionMiddleware({ mcApiUrl: CLOUD_IDENTIFIERS.GCP_EU }));
+app.use(
+  createSessionMiddleware({
+    audience: 'https://my-api-server.com',
+    issuer: CLOUD_IDENTIFIERS.GCP_EU,
+  })
+);
 app.use((request, response, next) => {
   // `request.session` contains the useful information
 });
 ```
 
 ### Middleware options
 
-- `issuer` (_string_): either a cloud identifier or a valid URL to the Merchant Center API Gateway. The cloud identifier maps to the Merchant Center API URL of the related [cloud region](https://docs.commercetools.com/custom-applications/main-concepts/api-gateway#cloud-regions).
+- `audience` (_string_): The public-facing URL of your API server. The value should only contain the origin URL (protocol, hostname, port), the request path is inferred from the incoming request.
```

Comment:
It's better to explicitly provide the `audience`, as the `req.hostname` only exists in express.js and might not even work in a serverless environment.

Reply:
True, did we release the prior version on latest or as a beta?

Reply:
Canary 😄